Chp9Lsn5/Chp9Lsn5/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp9Lsn5
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int i = UserInput();
+            int[] randomArray = { 2, 5, 2, 3, 4, 2, 8, 9, 7, 9 };
+
+            IsLargerThan(randomArray, i);
+        }
+
+        /// <summary>
+        /// Takes an index number from user.
+        /// </summary>
+        /// <returns>number 0-9</returns>
+        public static int UserInput()
+        {
+            int isInput = 10;
+            bool resolved = false;
+            Console.WriteLine("Please enter an index number (0-9). This will check if it is larger than both its neighbors in a number circle.");
+
+            while (resolved == false)
+            {
+                string input = Console.ReadLine();
+                bool isInt = System.Int32.TryParse(input, out isInput);
+
+                if (isInt && isInput > -1 && isInput < 10)
```

Comment:
Well done on the boolean logic!  Still, it "feels" better whenever a constant can be converted to zero.  For example:

    if (isInt && isInput >= 0 && isInput < 10)